backend/app/models/settings.py

```diff
@@ -23,13 +23,13 @@ class Settings(BaseSettings):
     arkg_mime_type: RdfMimeType = RdfMimeType.TURTLE
 
     anti_recommender_type: AntiRecommenderType = AntiRecommenderType.ARKG
-    openai_api_key: ApiKey | None = None
+    openai_api_key: SecretStr | None = None
     output_file_paths: frozenset[Path] = Field(
         default=frozenset(), validation_alias="output_file_names"
     )
     record_types: frozenset[RecordType] = frozenset()
-    supabase_url: str = ""
-    supabase_key: str = ""
+    supabase_url: AnyUrl = AnyUrl("https://nerdswipe.supabase.co")
```

Comment:
Just make this optional?

Reply:
`supabase_url` and `supabase_key` are now optional.